.eslintignore

```diff
@@ -1,3 +1,5 @@
 lib/types/api.ts
 lib/utils/fetcher.ts
+pages/_document.tsx
+components/common/ClientOnlyPortal.ts
```

Comment:
Not a fault of this PR but the file is missing a newline at the end

Reply:
Fixed in 9137767